src/main/java/com/google/sps/model/TasksClientImpl.java

```diff
@@ -61,6 +62,17 @@ private TasksClientImpl(Credential credential) {
     return taskLists != null ? taskLists : new ArrayList<>();
   }
 
+  @Override
+  public List<TaskList> listTaskLists(List<String> taskListTitles) throws IOException {
+    // returns null if no taskLists exist. Convert to empty list for ease.
+    List<TaskList> taskLists = listTaskLists();
+    taskLists = taskLists != null ? taskLists : new ArrayList<>();
```

Comment:
I think it's okay to trust that `listTaskLists()`won't return null (maybe add a comment to `listTaskLists` saying that it should not return null)

Reply:
I have changed the comment, thanks!

Reply:
`RESOLVED`